src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -184,19 +184,20 @@ export class QuickBar extends LitElement {
       this._itemFilter = newFilter;
     }
 
-    this._items = (this._commandMode ? this._commandItems : this._entityItems)
-      .filter(({ text, altText }) => {
-        const values = [text];
-        if (altText) {
-          values.push(altText);
-        }
-        return fuzzySequentialMatch(this._itemFilter.trimLeft(), values);
-      })
-      .sort((itemA, itemB) => compare(itemA.text, itemB.text));
+    this._items = this._commandMode ? this._commandItems : this._entityItems;
```

Comment:
put this in the else of the if below

Reply:
I'll do you one better and do the thing I was _intending_ to do there:

```
  this._items = this._commandMode ? this._commandItems : this._entityItems;

    if (this._itemFilter !== "") {
      this._items = fuzzyFilterSort<QuickBarItem>(
        this._itemFilter.trimLeft(),
        this._items
      );
    }
```

Reply:
Done